dockerfiles/haskell/README.md

```diff
@@ -0,0 +1,52 @@
+# Experimental Haskell language server
+
+![demonstration of hovers from the Haskell language server inside chrismwendt/minijava](https://cl.ly/1h3E2P2s2g2y/haskell.gif)
+
+*This GIF was created by browsing http://localhost:3080/github.com/chrismwendt/MiniJava@6cb615856cfcad0253c9588a40a5b8678df05349/-/blob/src/RegisterAllocator.hs with this Haskell language server enabled.*
+
+## Introduction
+
+This Dockerfile adds experimental Haskell language support for Sourcegraph.
+
+Thanks to the [haskell/haskell-ide-engine](https://github.com/haskell/haskell-ide-engine) project for providing the language server that's wrapped by `lsp-adapter` in this image.
+
+## How to add this language server to Sourcegraph
+
+*These steps are adapted from our documentation for [manually adding code intelligence to Sourcegraph server](https://about.sourcegraph.com/docs/code-intelligence/install-manual/).*
+
+1. Run the `sourcegarph/server` Docker image:
```

Comment:
See my comment on the other PR. I don’t think we should have a readme per dockerfile because it duplicates a lot of info.

We should have a single doc page for these steps, ideally on about.sourcegraph.com/docs and just link to that page from the roof readme

Reply:
:+1: I filed #59